src/types/composite/list.ts

```diff
@@ -17,6 +18,13 @@ type ListTypeConstructor = {
   new<T extends List<any>>(options: IListOptions): ListType<T>;
 };
 
+export const LIST_TYPE = Symbol.for("ssz/ListType");
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isListType<T extends List<any>=List<any>>(type: unknown): type is ListType<T> {
```

Comment:
same question as above, but i see that the type `ListType` already uses this `List<any>` so i guess i'm also wondering if we can find a way around the any's there too.  would using `unknown` type be possible (such as `List<unknown>`?  or maybe some other way?

Reply:
I'm copying the pattern below (in this file). I think we can revisit these `any`s in list.ts and vector.ts, but should probably do it separately to this PR.